pkg/aaparser/aaparser.go

```diff
@@ -26,7 +25,7 @@ func GetVersion() (int, error) {
 // LoadProfile runs `apparmor_parser -r` on a specified apparmor profile to
 // replace the profile.
 func LoadProfile(profilePath string) error {
-	_, err := cmd("", "-r", filepath.Dir(profilePath))
```

Comment:
The code originally was filepath.Base(profilePath) back when it was saving to /etc/apparmor.d/. Then when the save location was changed to /tmp it was changed to filePath.Dir, which currently calls apparmor_parser -r /tmp and loads in all of the unique-named apparmor profiles in /tmp (which is clearly wrong). 

Changing it to profilePath (which will call apparmor_parser -r /tmp/docker-default$hash) or even filepath.Abs(profilepath) which will call clean on profilePath, seems sensible to me.